clients/src/main/java/org/apache/kafka/clients/producer/KafkaProducer.java

```diff
@@ -1237,8 +1251,7 @@ private int partition(ProducerRecord<K, V> record, byte[] serializedKey, byte[]
         Integer partition = record.partition();
         return partition != null ?
                 partition :
-                partitioner.partition(
-                        record.topic(), record.key(), serializedKey, record.value(), serializedValue, cluster);
+                partitioner.partition(record.topic(), record.key(), serializedKey, record.value(), serializedValue, cluster);
```

Comment:
Let's avoid making whitespace changes like this, since they tend to make the change harder to read